lib/omnibus/licensing.rb

```diff
@@ -650,98 +693,99 @@ def collect_licenses_for(software)
-      "MS-PL",         # Microsoft Public License
-      "MS-RL",         # Microsoft Reciprocal License
-      "MIT",           # MIT license
-      "Motosoto",      # Motosoto License
-      "MPL-2.0",       # Mozilla Public License 2.0
-      "Multics",       # Multics License
-      "NASA-1.3",      # NASA Open Source Agreement 1.3
-      "NTP",           # NTP License
-      "Naumen",        # Naumen Public License
-      "NGPL",          # Nethack General Public License
-      "Nokia",         # Nokia Open Source License
-      "NPOSL-3.0",     # Non-Profit Open Software License 3.0
-      "OCLC-2.0",      # OCLC Research Public License 2.0
-      "OGTSL",         # Open Group Test Suite License
-      "OSL-3.0",       # Open Software License 3.0
-      "OPL-2.1",       # OSET Public License version 2.1
-      "PHP-3.0",       # PHP License 3.0
-      "PostgreSQL",    # The PostgreSQL License
-      "Python-2.0",    # Python License
-      "CNRI-Python",   # CNRI Python license
-      "QPL-1.0",       # Q Public License
-      "RPSL-1.0",      # RealNetworks Public Source License V1.0
-      "RPL-1.5",       # Reciprocal Public License 1.5
-      "RSCPL",         # Ricoh Source Code Public License
-      "OFL-1.1",       # SIL Open Font License 1.1
-      "SimPL-2.0",     # Simple Public License 2.0
-      "Sleepycat",     # Sleepycat License
-      "SPL-1.0",       # Sun Public License 1.0
-      "Watcom-1.0",    # Sybase Open Watcom Public License 1.0
-      "NCSA",          # University of Illinois/NCSA Open Source License
-      "UPL",           # Universal Permissive License
-      "VSL-1.0",       # Vovida Software License v. 1.0
-      "W3C",           # W3C License
-      "WXwindows",     # wxWindows Library License
-      "Xnet",          # X.Net License
-      "0BSD",          # Zero Clause BSD License
-      "ZPL-2.0",       # Zope Public License 2.0
-      "Zlib",          # zlib/libpng license
+      "AFL-3.0" => "https://opensource.org/licenses/AFL-3.0",             # Academic Free License 3.0
+      "AGPL-3.0" => "https://opensource.org/licenses/AGPL-3.0",           # Affero General Public License
+      "APL-1.0" => "https://opensource.org/licenses/APL-1.0",             # Adaptive Public License
+      "Apache-2.0" => "https://opensource.org/licenses/Apache-2.0",       # Apache License 2.0
+      "APSL-2.0" => "https://opensource.org/licenses/APSL-2.0",           # Apple Public Source License
+      "Artistic-2.0" => "https://opensource.org/licenses/Artistic-2.0",   # Artistic license 2.0
+      "AAL" => "https://opensource.org/licenses/AALAAL",                  # Attribution Assurance Licenses
+      "BSD-3-Clause" => "https://opensource.org/licenses/BSD-3-Clause",   # BSD 3-Clause "New" or "Revised" License
+      "BSD-2-Clause" => "https://opensource.org/licenses/BSD-2-Clause",   # BSD 2-Clause "Simplified" or "FreeBSD" License
+      "BSD-1-Clause"  => "https://opensource.org/licenses/BSD-1-Clause",  # BSD 1-Clause
```

Comment:
Linter is complaining because of the extra space:

```suggestion
      "BSD-1-Clause" => "https://opensource.org/licenses/BSD-1-Clause",   # BSD 1-Clause
```